Cubical/Cohomology/EilenbergMacLane/EilenbergSteenrod.agda

```diff
@@ -0,0 +1,282 @@
+{-# OPTIONS --safe --experimental-lossy-unification #-}
+
+module Cubical.Cohomology.EilenbergMacLane.EilenbergSteenrod where
+
+open import Cubical.Homotopy.EilenbergMacLane.GroupStructure
+open import Cubical.Homotopy.EilenbergMacLane.Base
+open import Cubical.Homotopy.EilenbergMacLane.Properties
+open import Cubical.Homotopy.EilenbergSteenrod
+
+open import Cubical.Cohomology.EilenbergMacLane.Base
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Pointed.Homogeneous
+open import Cubical.Foundations.GroupoidLaws
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+
+open import Cubical.Algebra.Group
+open import Cubical.Algebra.Group.Morphisms
+open import Cubical.Algebra.Group.MorphismProperties
+open import Cubical.Algebra.AbGroup
+open import Cubical.Algebra.AbGroup.Instances.Unit
+open import Cubical.Algebra.Group.Instances.Unit
+
+open import Cubical.HITs.Wedge
+open import Cubical.HITs.Pushout
+open import Cubical.HITs.Susp
+open import Cubical.HITs.Truncation as T
+open import Cubical.HITs.SetTruncation as ST
+open import Cubical.HITs.PropositionalTruncation as PT
+
+open import Cubical.Data.Empty as ⊥
+open import Cubical.Data.Nat
+open import Cubical.Data.Bool
+open import Cubical.Data.Sigma
+open import Cubical.Data.Int
+open import Cubical.Data.Unit
+
+open Iso
+open coHomTheory
+
+-- This module verifies the (reduced) Eilenberg-Steenrod axioms for
+-- the following ℤ-indexed cohomology theory
+
+coHomRedℤ : ∀ {ℓ ℓ'} → AbGroup ℓ' → ℤ → Pointed ℓ → AbGroup (ℓ-max ℓ ℓ')
+coHomRedℤ G (pos n) A = coHomRedGr n G A
+coHomRedℤ G (negsuc n) A = UnitAbGroup
+
+module _ {ℓ ℓ'} {G : AbGroup ℓ} where
```

Comment:
Maybe this module should be called ```coHomRed``` or something like that? You use that as a suffix sometimes below...